pyadaptivecard/__init__.py

```diff
@@ -279,7 +287,7 @@ def send(self):
                 }
             ]
         }
-        response = requests.post(self.url, json=message_body, headers=header)
+        response = requests.post(self.url, json=message_body, headers=header, proxies={"https": self._proxy})
```

Comment:
Should be set only if self.proxy is not None? I'm not sure proxies={"https":None} works

Reply:
I'll add commit after that before next release